app/src/main/AndroidManifest.xml

```diff
@@ -67,16 +67,7 @@
     <activity
       android:name=".splash.SplashActivity"
       android:screenOrientation="portrait"
-      android:theme="@style/SplashScreenTheme">
-      <intent-filter>
-        <action android:name="android.intent.action.MAIN" />
```

Comment:
Make splash screen as launcher. And mention steps for testing the `GridAutoFitLayoutManagerTestActivity ` in description like changing the Launcher activity.

Reply:
Done.